src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -1893,18 +1897,18 @@ def test_storage_account_or_policy(self, resource_group, source_account, destina
                 .get_output_in_json()
             json.dump(policy, f)
         self.kwargs['policy'] = policy_file
-        self.cmd('storage account or-policy create -g {rg} -n {src_sc} -p @"{policy}"')\
-            .assert_with_checks(JMESPathCheck('type', "Microsoft.Storage/storageAccounts/objectReplicationPolicies")) \
-            .assert_with_checks(JMESPathCheck('sourceAccount', source_account)) \
-            .assert_with_checks(JMESPathCheck('destinationAccount', destination_account)) \
-            .assert_with_checks(JMESPathCheck('rules[0].sourceContainer', src_container)) \
-            .assert_with_checks(JMESPathCheck('rules[0].destinationContainer', dest_container)) \
-            .assert_with_checks(JMESPathCheck('rules[0].filters.minCreationTime', '2020-02-19T16:05:00Z'))
+        result = self.cmd('storage account or-policy create -g {rg} -n {src_sc} -p @"{policy}"').get_output_in_json()
+        self.assertEqual(result['type'], "Microsoft.Storage/storageAccounts/objectReplicationPolicies")
+        self.assertIn(self.kwargs['src_sc'], result['sourceAccount'])
+        self.assertIn(self.kwargs['dest_sc'], result['destinationAccount'])
```

Comment:
```suggestion
        self.assertEqual(self.kwargs['src_sc_id'], result['sourceAccount'])
        self.assertEqual(self.kwargs['dest_sc_id'], result['destinationAccount'])
```
May I know why use `assertIn` instead of `assertEqual`?

Reply:
As there is a potential service change, which will change to return resource id for PU, but currently not as you can see in https://msazure.visualstudio.com/DefaultCollection/One/_git/Storage-SRP/pullrequest/4515226.

We could change the assert after service team fix, but as it is JEDI requirement, I don't want to block CLI support.